### PR TITLE
Aeb pre jor

### PR DESCRIPTION
**Pull request by astropixie**

### CODE REVIEW

File: body.tex [hunk 1]
```diff
@@ -48,80 +50,133 @@ \section{Rubin Operations Leadership and Annual Reporting}
...
+This may be particularly the case for technical managers or others in leadership roles.
 In general, we strive to minimize the fraction of effort which is devoted to \gls{LOE} activities and favor those which are more directly accountable.
 However, in certain cases such as operations and maintenance of telescope and facility systems, pipelines or other systems, \gls{LOE} is perfectly acceptable.
-
-As an example, a first-order estimate is that developers will spent 30\% of their time on \gls{LOE} type activities, and the remaining 70\% of their effort is tracked against concrete deliverables.
-Technical staff in Chile at the summit facility may spend a much more significant fraction of time as  \gls{LOE}.
-However, as above, we generally aspire to minimize the fraction of \gls{LOE} for development activity.
+Technical staff in Chile at the summit facility may spend a much more significant fraction of time as \gls{LOE}.
+As an example, a first-order estimate is that developers will spent 30\% of their time on \gls{LOE} type activities, and the remaining 70\% of their effort is planned and tracked against well-defined deliverables.
```
Comment (womullan) [inline body.tex:149]: That seems very optimistic for operations where most work is emergent - fix this major issue. Its not planned. Who came up with this 70/30 ?   That is more like construction.

File: body.tex [hunk 2]
```diff
@@ -203,22 +260,23 @@ \section{Long Term Planning}
...
 Level 3 milestones are defined for use within Departments and not required to go under project change control, but properly adhering to the plan is important: your colleagues in other teams will use these milestones to align their schedules with yours, so they rely on you to be accurate.
 
-Epics should work to achieve milestones i.e. they may be blocking issues on the milestones.
+Epics should work to achieve milestones, i.e., they may be blocking issues on the milestones.
 When a detailed description of work for a given \gls{epic} is known, it is described in \gls{JIRA}.
 It should then be assigned to the appropriate \glspl{cycle}.
 
-Progress is tracked toward achieving milestones in Smartsheet by monitoring completed story points in \gls{JIRA} epics and rolling up the total progress.
+Progress is tracked toward achieving milestones in Smartsheet by monitoring completed story points on linked issues in \gls{JIRA} epics and rolling up the total progress.
 To ensure success, \gls{JIRA} epics must be completely detailed out prior to a full 6-month cycle and total effort should be estimated out for an entire fiscal year of effort, as detailed below.
```
Comment (womullan) [inline body.tex:272]: and all milestones should appear in Jira which is the source of truth.

File: body.tex [hunk 3]
```diff
@@ -300,29 +360,30 @@ \subsubsection{Defining Epics} \label{sec:planning-epics}
...
 \item The \texttt{label} field is set to identify the fiscal year during which the work will be done. Examples are FY22 or FY23.
 
 \end{itemize}
 The fields above are required to have values entered because they define the connection to Smartsheet where effort-tracking for the full project is done. Other fields in the epic can also be filled in as needed.
 
-Be aware that: (Amanda: I don't think all of the following bullets are true or necessary in the Operations non-EVMS era.  Should clarify with Cathy and Wil.)
+Be aware that:
 
 \begin{itemize}
 \item An \gls{epic} may only be assigned to a single \gls{cycle}.
```
Comment (womullan) [inline body.tex:373]: We are not really doign cycles in ops and not cleanly like this. The epics have been event driven.

File: body.tex [hunk 4]
```diff
@@ -352,9 +413,10 @@ \subsubsection{Bucket Epics} \label{sec:bucket}
 As such, we acknowledge that they are necessary, but seek to minimize the fraction of our resources assigned to them.
 If more than a relatively small fraction of the work for a \gls{cycle} is assigned to bucket \glspl{epic}, please consider whether this is really necessary and appropriate.
 
-Be aware that even bucket \glspl{epic} must be assigned to a specific \emph{leaf} \gls{element} of the \gls{WBS}.
-That is, it is not in general possible to define an \gls{epic} which handles bug reports or emergent feature requests across the whole of the codebase unless a specific \gls{WBS} leaf \gls{element} is devoted to maintenance activities of this type.
-Instead, it may be necessary to define a different bucket \gls{epic} for each leaf of the \gls{WBS} tree.
+% I think this is a tie to the Construction EVMS and is not needed here.
```
Comment (womullan) [inline body.tex:416]: At some point you may like this to know which WBS we spend more on .. but indeed its like this construction for EVMS.

File: body.tex [hunk 5]
```diff
@@ -393,10 +466,44 @@ \subsection{Defining Stories}
...
+serviced promptly.
+
+In some cases, the technical manager responsible for a given repository
+is obvious, and they can be expected to take the lead on handling
+tickets. Often, this is not the case: repositories regularly span team
+boundaries. Work together to ensure that all tickets are handled.
+
+\subsubsection{Emergent Work}
+\label{emergent-work}
+
```
Comment (womullan) [inline body.tex:503]: Again, this is probably more the norm in operations.

File: body.tex [hunk 6]
```diff
@@ -490,35 +597,10 @@ \subsubsection{Completing the Work}
 \item (If applicable, i.e. it is not a bucket, as defined in \S\ref{sec:bucket}) its concrete deliverable has been achieved.
 \end{enumerate}
 
+``Bucket'' epics should be closed at the end of the time box (i.e. end of half fiscal year and end of fiscal year).
```
Comment (womullan) [inline body.tex:600]: We may also assign buckets to a specific milestones and close them when the milestones is achieved e.g. DR1 Bucket Epic or  DP0.2 Bucket epic.

File: body.tex [hunk 7]
```diff
@@ -490,35 +597,10 @@ \subsubsection{Completing the Work}
...
 Note that it is not permitted to close an \gls{epic} without defining at least one \gls{story} within it.
 Empty \glspl{epic} can never be completed.
 
-\subsection{Handling Bugs \& Emergent Work}
```
Comment (womullan) [inline body.tex:496]: if this goes out perhaps we should refer to developer.io ??  https://developer.lsst.io/work/flow.html

File: body.tex [hunk 8]
```diff
@@ -530,7 +612,7 @@ \subsection{Jira Maintenance}
 It is required that the \texttt{Team} field be set to the appropriate team (\jira{RFC-145}).
 This indicates which manager is responsible for \gls{seeing} that the work is completed successfully.
 Available teams, and the associated managers, are listed in the \href{https://developer.lsst.io/processes/jira_agile.html}{Developer Guide}; generally speaking, they align with the the work breakdown structure described in \S\ref{sec:wbs}.
-Where there is uncertainty about which team should be responsible for a particular ticket, the “System Management” team may be used to indicate that the \gls{DM} \gls{Project Manager} is responsible for assigning the work.
+Where there is uncertainty about which team should be responsible for a particular ticket, the “Data Production Management” team may be used to indicate that the \gls{DM} \gls{Project Manager} is responsible for assigning the work.
```
Comment (womullan) [inline body.tex:615]: I guess there is no DM project manager in ops in charge of Data Production - AD of DP ?

**Review by womullan — CHANGES_REQUESTED** [review]

File: body.tex (hunk 1 above)
Comment (astropixie) [reply body.tex:149]: Yes, the 70/30 is a carry over assumption from Construction. What do you suggest we include here for DP and SF folks?

File: body.tex (hunk 3 above)
Comment (astropixie) [reply body.tex:373]: Do you plan to move into a regime of cycle-driven work and construction ramps down?  That was an assumption I was working from.  We at least have the annual POP cycle that we will work within.

File: body.tex (hunk 8 above)
Comment (astropixie) [reply body.tex:615]:
```suggestion
Where there is uncertainty about which team should be responsible for a particular ticket, the “Data Production Management” team may be used to indicate that the \gls{AD} of  \gls{Data Production} is responsible for assigning the work.
```

File: body.tex (hunk 7 above)
Comment (astropixie) [reply body.tex:496]: This section can be included if you want, but with a note that it is for DP and SF, not ROO or REO (at this time).

File: body.tex (hunk 2 above)
Comment (astropixie) [reply body.tex:272]:
```suggestion
To ensure success, \gls{JIRA} epics must be completely detailed out prior to a full 6-month cycle and total effort should be estimated out for an entire fiscal year of effort, as detailed below.
All milestones should appear in \gls{JIRA} with a milestone issue type as the source of truth. 
```

**Review by astropixie — COMMENTED** [review]
Thanks for your comments, Wil!  I asked for a couple suggestions to modify text.

File: body.tex (hunk 1 above)
Comment (womullan) [reply body.tex:149]: 50 50 but we can update later

File: body.tex (hunk 7 above)
Comment (womullan) [reply body.tex:496]: Refer to developer.io -- and we should agree how this is done for all..

**Review by womullan — APPROVED** [review]

File: body.tex [hunk 9]
```diff
@@ -530,7 +613,7 @@ \subsection{Jira Maintenance}
 It is required that the \texttt{Team} field be set to the appropriate team (\jira{RFC-145}).
 This indicates which manager is responsible for \gls{seeing} that the work is completed successfully.
 Available teams, and the associated managers, are listed in the \href{https://developer.lsst.io/processes/jira_agile.html}{Developer Guide}; generally speaking, they align with the the work breakdown structure described in \S\ref{sec:wbs}.
-Where there is uncertainty about which team should be responsible for a particular ticket, the “System Management” team may be used to indicate that the \gls{DM} \gls{Project Manager} is responsible for assigning the work.
+Where there is uncertainty about which team should be responsible for a particular ticket, the “Data Production Management” team may be used to indicate that the \gls{AD} of  \gls{Data Production} is responsible for assigning the work.
```
Comment (astropixie) [inline body.tex:616]:
```suggestion
Where there is uncertainty about which team should be responsible for a particular ticket, the “Data Production Management” team may be used to indicate that the \gls{AD} of Data Production is responsible for assigning the work.
```

File: body.tex [hunk 10]
```diff
@@ -1,42 +1,44 @@
 \section{Introduction}
 
+This document provides a guide to the \VRO\ approach to work management and annual planning.
+See the operations proposal \citeds{RDO-018} for a description of the full scope and high level goals of the program.
```
Comment (astropixie) [inline body.tex:4]:
```suggestion
See the operations proposal RDO-018 for a description of the full scope and high level goals of the program.
```

**Pull request merged**